### ISSUE TITLE
Package postinstall assumes npm is present

### ISSUE THREAD

**Issue by meltingice** [Shopify/polaris#123]
# Issue summary

When installing polaris, there is a `postinstall` script that runs `npm rebuild node-sass`, which assumes that npm is installed on the system. While this may have been true in the past, we now have yarn as well. This is especially an issue when deploying to Heroku with an app that uses yarn, because it will only install yarn and not npm on the dynos causing deployment failure.

## Expected behavior

Post-install should detect whether yarn or npm is in use and use the appropriate package manager. The equivalent action in yarn would be:

```
yarn remove node-sass
yarn add node-sass
```

## Actual behavior

```
error /tmp/build_9d1a0d46e697ed8afc5b84747749baac/node_modules/@shopify/polaris: Command failed.
Exit code: 127
Command: sh
Arguments: -c npm rebuild node-sass
Directory: /tmp/build_9d1a0d46e697ed8afc5b84747749baac/node_modules/@shopify/polaris
Output:
sh: 1: npm: not found
```

## Steps to Reproduce the Problem

1. Install polaris on a system using yarn without npm installed.

## Specifications

- Polaris version: 1.1.0

**Comment by lemonmade** [comment]
Thanks for the issue, @meltingice. We'll get this removed for the next version.

**Comment by dfmcphee** [comment]
This was resolved in 1.2.0
